<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zoomWindow</class>
 <widget class="QWidget" name="zoomWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Grabber - Image</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/images/icon.ico</normaloff>:/images/icon.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
     <item>
      <widget class="QLabel" name="labelFormTags">
       <property name="text">
        <string>&lt;b&gt;Tags&lt;/b&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelTags">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonDetails">
       <property name="text">
        <string>Plus d'infos</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonsLayout">
     <item>
      <widget class="QPushButton" name="buttonSave">
       <property name="text">
        <string>Enregistrer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSaveNQuit">
       <property name="text">
        <string>Enregistrer et fermer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOpen">
       <property name="text">
        <string>Ouvrir le dossier de destination</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSaveAs">
       <property name="text">
        <string>Enregistrer sous...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonSave</sender>
   <signal>clicked()</signal>
   <receiver>zoomWindow</receiver>
   <slot>saveImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>450</y>
    </hint>
    <hint type="destinationlabel">
     <x>68</x>
     <y>472</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSaveNQuit</sender>
   <signal>clicked()</signal>
   <receiver>zoomWindow</receiver>
   <slot>saveNQuit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>447</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>469</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOpen</sender>
   <signal>clicked()</signal>
   <receiver>zoomWindow</receiver>
   <slot>openSaveDir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>442</x>
     <y>455</y>
    </hint>
    <hint type="destinationlabel">
     <x>471</x>
     <y>469</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSaveAs</sender>
   <signal>clicked()</signal>
   <receiver>zoomWindow</receiver>
   <slot>saveImageAs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>658</x>
     <y>455</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>468</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>labelTags</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>zoomWindow</receiver>
   <slot>contextMenu(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>548</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>596</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>labelTags</sender>
   <signal>linkHovered(QString)</signal>
   <receiver>zoomWindow</receiver>
   <slot>linkHovered(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>labelTags</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>zoomWindow</receiver>
   <slot>openUrl(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>saveImage()</slot>
  <slot>saveNQuit()</slot>
  <slot>openSaveDir()</slot>
  <slot>saveImageAs()</slot>
  <slot>linkHovered(QString)</slot>
  <slot>openUrl(QString)</slot>
  <slot>contextMenu(QPoint)</slot>
 </slots>
</ui>
